tests/php/sync/test_class.jetpack-sync-themes.php

```diff
@@ -34,6 +34,65 @@ function theme_info() {
 class WP_Test_Jetpack_Sync_Themes extends WP_Test_Jetpack_Sync_Base {
 	protected $theme;
 
+	/**
+	 * Dummy Themes.
+	 *
+	 * @var string[]
+	 */
+	protected $themes = array(
+		'theme-file-sync-parent',
+		'theme-file-sync-child',
+	);
```

Comment:
You don't seem to ever use this. Why not make it static, and then use it in `setUpBeforeClass()` and `tearDownAfterClass()` below?

Reply:
:thumbsup: had intentions to use it but didn't get around to it.

Reply:
Updated.